scripts/c2r_script.py

```diff
@@ -278,12 +344,26 @@ def generate_report_message(highest_status):
     message = ""
     alert = False
 
-    if STATUS_CODE[highest_status] <= STATUS_CODE["WARNING"]:
-        message = (
-            "No problems found. The system was converted successfully. Please,"
-            " reboot your system at your earliest convenience to make sure that"
-            " the system is using the RHEL Kernel."
-        )
+    conversion_successful_msg = (
+        "No problems found. The system was converted successfully. Please,"
+        " reboot your system at your earliest convenience to make sure that"
+        " the system is using the RHEL Kernel."
+    )
+
+    if STATUS_CODE[highest_status] < STATUS_CODE["WARNING"]:
+        if IS_CONVERSION:
+            message = conversion_successful_msg
+        elif IS_ANALYSIS:
+            message = "No problems found. The system is ready for conversion."
```

Comment:
We only have two states in convert2rhel for now.. No need to explicitly use an `elif`


```suggestion
        message = conversion_successful_msg if IS_CONVERSION else "No problems found. The system is ready for conversion."
```

Reply:
Yep, I had it there before I introduced the check for valid values in main function :) `else` makes more sense now